### PR TITLE
Show public key and type not just address

### PR DESCRIPTION
**Pull request by jarrydallison**
## Description

Shows the public key type and key value instead of the address

closes: #198 

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [X] Targeted PR against correct branch
- [X] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [X] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer

### CODE REVIEW

**Comment by jarrydallison** [comment]
Updated to show public key type and value in a single field.
Screen Shot 2022-01-06 at 11 02 39 AM

**Comment by jarrydallison** [comment]
I could also change from "Key" to "Value" if that makes more sense?

File: src/Pages/Accounts/Components/AccountSpotlight.js [hunk 1]
```diff
@@ -42,8 +44,12 @@ const AccountSpotlight = () => {
     { title: 'Account Number', value: accountNumber },
     {
       title: 'Public Key',
-      value: matches ? maxLength(publicKey, 20, 3) : publicKey,
+      value: [
+        [`Type:`, `${publicKeyType}`],
+        [`Key:`, `${matchesList ? maxLength(publicKey, 20, 3) : publicKey}`],
```
Comment (jarrydallison) [inline src/Pages/Accounts/Components/AccountSpotlight.js:49]: Should this title here be "Value" instead of "Key"?

Comment (webbushka) [reply src/Pages/Accounts/Components/AccountSpotlight.js:49]: @minxylynx what are your thoughts? do you care what the label says?

**Review by webbushka — DISMISSED** [review]

**Review by webbushka — APPROVED** [review]

**Pull request merged**